src/Microsoft.AspNetCore.Razor.Sdk/build/netstandard2.0/Razor.Sdk.CurrentVersion.targets

```diff
@@ -72,8 +72,10 @@ Copyright (c) .NET Foundation. All rights reserved.
 
     <!-- Default to on if MvcRazorCompileOnPublish isn't set for some reason -->
     <RazorCompileOnPublish Condition="'$(RazorCompileOnPublish)'==''">true</RazorCompileOnPublish>
-    
-    <UseRazorBuildServer Condition="'$(UseRazorBuildServer)'==''">false</UseRazorBuildServer>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(UseRazorBuildServer)'==''">
+    <UseRazorBuildServer>false</UseRazorBuildServer>
```

Comment:
revert this, it was better bfore

Reply:
Really? It was under a property group with `Condition="'$(RazorCompileOnPublish)'==''`. What about the cases when it isn't satisfied?

Reply:
Btw, why doesn't this live in the props file?

Reply:
I was just following the same pattern as `RazorCompileOnBuild` which is also set in the same file.

Reply:
Ah gotcha. Yeah the right fix for this is to go in the .props file like:
```
<PropertyGroup>
<UseRazorBuildServer Condition="'$(UseRazorBuildServer)'==''">false</UseRazorBuildServer>
</PropertyGroup>
```

You can put it inside another propertygroup as long as that propertygroup doesn't have a condition

Avoid putting conditions on propertygroups unless you have a real need for it.